### DISCUSSION TITLE
I fckd up my tablet

### DISCUSSION THREAD

**Discussion by vetyalmao** [ppy/osu#15493] [Q&A] [answered]
i installed the new update and it fckd up my tablet. its like my cursor is in 30fps and everything else is fine. this didnt happened before this update, i think.

**Comment by frenzibyte** [comment]
I'm not sure how we can help witout any actionable details. 

 - Did you downgrade to the older game version and check if it does work fine?
 - Can you state exactly which version the tablet input started to get processed slower (i.e. lagging)
 - Are you referring to the built-in tablet driver or an external driver you're running in your machine?

**Reply by vetyalmao** [reply]
> I'm not sure how we can help witout any actionable details.
> 
> * Did you downgrade to the older game version and check if it does work fine?
> * Can you state exactly which version the tablet input started to get processed slower (i.e. lagging)
> * Are you referring to the built-in tablet driver or an external driver you're running in your machine?

1. older version doesnt fix it
2.idk what do you mean. the lagging is like its missing some cursortrail parts. but the cursor is lagging too.
3.opentabletdriver, i tried wacom, hawku drivers too. didnt fix it.

**Comment by peppy** [comment, marked as answer]
Sounds like your window isn't in focus. Try restarting osu! and your PC.

**Reply by vetyalmao** [reply]
> Sounds like your window isn't in focus. Try restarting osu! and your PC.

already restarted osu, and pc 101 times.

**Reply by vetyalmao** [reply]
> Sounds like your window isn't in focus. Try restarting osu! and your PC.

what do you mean window isnt in focus? its on fullscreen, 240hz monitor, windows 11

**Reply by integerrr** [reply]
screenshot the fps display and upload your logs

**Reply by vetyalmao** [reply]
> screenshot the fps display and upload your logs

i dont know how to do these :(

**Reply by integerrr** [reply]
1. fps display
<img width="176" alt="image" src="https://user-images.githubusercontent.com/7279624/140621218-e3bb06d5-d469-424e-829d-272ebd518636.png"> turn it on with ctrl-F11 if its not on


2. logs
Read #15188

**Reply by vetyalmao** [reply]
> 1. fps display
>    <img alt="image" width="176" src="https://user-images.githubusercontent.com/7279624/140621218-e3bb06d5-d469-424e-829d-272ebd518636.png"> turn it on with ctrl-F11 if its not on
> 2. logs
>    Read [Please provide logs to help us troubleshoot your problems #15188](https://github.com/ppy/osu/discussions/15188)
[session.log](https://github.com/ppy/osu/files/7492124/session.log)
[update_success.log](https://github.com/ppy/osu/files/7492125/update_success.log)
[runtime.log](https://github.com/ppy/osu/files/7492126/runtime.log)
 i dont know how to screenshot the fps. it doesnt show me that. it shows me a large rectangle with 20ms on top and 0ms on bottom.
![image](https://user-images.githubusercontent.com/85455663/140642434-746379b0-5f59-4e0f-b909-641ae53b1e71.png) like this

**Reply by integerrr** [reply]
You're asking in the wrong place, osu!stable issues goes [here](https://github.com/ppy/osu-stable-issues/issues)
